DocumentationGuide.md

```diff
@@ -0,0 +1,184 @@
+<img src="External/ReadMeImages/MRTK_Logo_Rev.png">
+
+# Documentation Guidelines
+
+This document outlines the documentation guidelines and standards for the Mixed Reality Toolkit (MRTK). Herein you will find the standards for the following formsfs of the MRTK documentation:
+
+- [Source](#source-documentation)
+- [Conceptual](#conceptual-documentation)
+- [Design](#design-documentation)
+- [Performance Notes](#performance-notes)
+- [Breaking Changes](#breaking-changes)
+
+---
+
+## Source Documentation
+API documentation will be generated automatically from the MRTK source files. To facilitate this, source files are required to contain the following: 
+
+- [Class, Struct, Enum Summary Blocks](#class-struct-enum-summary-blocks)
+- [Property, Method, Event Summary Blocks](#property-method-event-summary-blocks)
+- [Feature Introduction Version and Dependencies](#feature-introductuin-version-and-dependencies)
+- [Serialized Fields](#serialized-fields)
+- [Enumeration Values](#enumeration-values)
+
+In addition to the above, the code should be well commented to allow for maintenance, bug fixes and ease of customization. 
+
+### Class, Struct, Enum Summary Blocks
+If a class, struct or enum is being added to the MRTK, it's purpose must be described. This is to take the form of a summary block above the class.
+
+```
+    /// <summary>
+    /// AudioOccluder implements IAudioInfluencer to provide an occlusion effect.
+    /// </summary>
+```
+
+If there are any class level dependencies, they should be documented in a remarks block, immediately below the summary.
+
+```
+    /// <remarks>
+    /// Ensure that all sound emitting objects have an attached AudioInfluencerController. 
+    /// Failing to do so will result in the desired effect not being applied to the sound.
+    /// </remarks>
+```
+
+Pull Requests submitted without summaries for classes, strutures or enums will not be approved.
+
+### Property, Method, Event Summary Blocks
+As with classes, structures and enums, PMEs (and fields) are to be documented with summary blocks, regardless of code visibility (public, private, protected and internal). The documentation generation tool is responsible for filtering out and publishing only the public and protected features.
```

Comment:
Explain what a PME is before using the acronym. Always explain terms before their use.

Reply:
Thanks. That was a casualty of a change I made at the last minute....